catalog/core/catalog-core-commands/src/main/java/org/codice/ddf/commands/catalog/DuplicateCommands.java

```diff
@@ -155,23 +182,39 @@ protected void duplicateInBatches(
               0L,
               TimeUnit.MILLISECONDS,
               blockingQueue,
-              StandardThreadFactoryBuilder.newThreadFactory("duplicateCommandsThread"),
```

Comment:
❓ Why was this thread factory removed?

Reply:
Most likely just  overlooked when merge conflicts were resolved, will fix.